Userland/Libraries/LibGUI/UndoStack.cpp

```diff
@@ -111,14 +111,14 @@ void UndoStack::clear()
         on_state_change();
 }
 
-Optional<DeprecatedString> UndoStack::undo_action_text() const
+Optional<ErrorOr<String>> UndoStack::undo_action_text() const
```

Comment:
`Optional<ErrorOr<String>>` feels like the wrong way around. It should really be `ErrorOr<Optional<String>>` so you can directly `TRY()` the function (this seems to be how this is handled elsewhere).